Ports/build_all.sh

```diff
@@ -49,38 +66,33 @@ for file in *; do
                 popd > /dev/null
                 continue
             fi
-
-            if [ "$clean" == true ]; then
-                if [ "$verbose" == true ]; then
-                    ./package.sh clean_all
-                else
-                    ./package.sh clean_all > /dev/null 2>&1
-                fi
-            fi
             if [ "$verbose" == true ]; then
                 if ./package.sh; then
-                    echo "Built ${port}."
+                    echo "Built $port."
```

Comment:
Please keep the curly braces. Same for the other three lines below this that have been changed as well (and, I guess, all the other added usages of variables).